test_runner/src/main/kotlin/ftl/args/ValidateIosArgs.kt

```diff
@@ -13,6 +13,13 @@ fun IosArgs.validate() = apply {
     assertTestFiles()
     checkResultsDirUnique()
     assertAdditionalIpas()
+    assertGameloop()
+}
+
+fun IosArgs.assertGameloop() {
+    /*if (scenarioNumbers.isNotEmpty() && (type == null || type != Type.GAMELOOP))
+        throw FlankConfigurationError("Scenario numbers defined but Type is not Game-loop.")*/
```

Comment:
if this commented code needed?

Reply:
this is a draft pull request. changes will come.